ApplicationRequirements.txt

```diff
@@ -3,13 +3,28 @@
 	"Password" Text Box
 	"Sign-in" Button
 		-> Takes to "Home" Screen if correct
+			-> Takes to the Driver variant of the "Home" Screen if account is registered to a driver
 		-> Redirects back to "Log-in" Screen if incorrect
 			-> Notifies user with red text
 	"Create New Account" Button
 		-> Takes to "Create New Account" Screen
 		
-"Home" Screen
-
+"Home" Screen - General
+	"Create New Request" Button
+		-> Takes to "Create New Request" Screen
+	"My Requests" Button
+		-> Takes to "My Requests" Screen
+		
+"Home" Screen - Driver
+	"Create New Request" Button
+		-> Takes to "Create New Request" Screen
+	"My Requests" Button
+		-> Takes to "My Requests" Screen
+	"Requests Available for Pickup" Button
+		-> Takes to "Requests Available for Pickup" Screen
+	"My Deliveries" Button
+		-> Takes to "My Deliveries" Screen
+		
 "Create New Account" Screen
```

Comment:
I think we should have 2 separate screens for creating an account. One of the user and one for the driver because we will ask them different information when they sign up.

Reply:
You are completely right about that. I completely forgot about signing drivers up. Do you think maybe it'd be best to create one out of an already existing user? Maybe just a button on the home screen that says "Become a Driver" or something and it sends to a page requesting the car info and such?

Reply:
Oh I didnt think of that, I really like this idea! We can have one sign up page since initially we need a username, email, password etc for both users and drivers. But then after a user registers they can become a driver on the site by clicking a button and providing more information, this way they can still make order requests and use the site as a user!